.circleci/config.yml

```diff
@@ -206,84 +216,101 @@ jobs:
             bundle add solidus --git "file://$(ruby -e"puts File.expand_path ENV['CIRCLE_WORKING_DIRECTORY']")"
             export LIB_NAME=set # dummy requireable file
             bundle exec rake -rrails -rspree/testing_support/extension_rake -e'Rake::Task["extension:test_app"].invoke'
-
-  postgres:
-    executor: postgres
+  
+  test_solidus:
+    parameters:
+      ex:
+        type: string
+        default: postgres
```

Comment:
Should this parameter be named `database` instead? 
Probably just me but I had to read twice in order to understand the reference to `executors`